src/test/scala/org/broadinstitute/dsde/firecloud/MockServers.scala

```diff
@@ -136,6 +184,47 @@ object MockServers {
           )
           .withStatusCode(OK.intValue)
       )
+
+    // entity-level responses
+
+    MockServers.workspaceServer
+      .when(
+        request()
+          .withMethod("POST")
+          .withPath(s"/workspaces/${mockSampleValid.wsNamespace}/${mockSampleValid.wsName}/entities")
+          .withBody(mockSampleValid.toJson.compactPrint)
+          .withCookies(cookie)
+      ).respond(
+        response()
+          .withHeaders(header)
+          .withStatusCode(Created.intValue)
+      )
+
+    MockServers.workspaceServer
+      .when(
+        request()
+          .withMethod("POST")
+          .withPath(s"/workspaces/${mockPairValid.wsNamespace}/${mockPairValid.wsName}/entities")
+          .withBody(mockPairValid.toJson.compactPrint)
+          .withCookies(cookie)
+      ).respond(
+        response()
+          .withHeaders(header)
+          .withStatusCode(Created.intValue)
+      )
+
+    MockServers.workspaceServer
+      .when(
+        request()
+          .withMethod("POST")
+          .withPath(s"/workspaces/${mockSampleConflict.wsNamespace}/${mockSampleConflict.wsName}/entities")
+          .withBody(mockSampleConflict.toJson.compactPrint)
+          .withCookies(cookie)
+      ).respond(
+        response()
+          .withHeaders(header)
+          .withStatusCode(Conflict.intValue)
+      )
```

Comment:
In Rawls we had the issue of having to implement mock functionality for every service call we provide, which was a lot of duplicated effort - this was instead changed to using the existing functionality with an in-memory database.  I'd like to see if it would be possible to also use Rawls as a mock in-memory service with in-memory database so that we could avoid having to mock all Rawls functionality here if that makes sense.